src/testing

```diff
@@ -0,0 +1,111 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+
+class Employee {
+public:
+    string id;
+    string name;
+
+    Employee(string empId, string empName) : id(empId), name(empName) {}
+};
+
+class EmployeeManager {
+private:
+    vector<Employee> employees;
```

Comment:
We already discussed that we are going to use LinkList Insted of any pre-implemented STL library.